tools/tpm2_unseal.c

```diff
@@ -93,13 +93,14 @@ bool unseal_and_save(TSS2_SYS_CONTEXT *sapi_context) {
         return false;
     }
 
+    tpm2_tool_output("sensitive: |\n");
+    tpm2_util_hexdump(outData.buffer, outData.size, false, true);
```

Comment:
Same for using hexdump here, this just makes users to need a tool like `xxd` to get the original sealed data for no good reasons IMHO. And the same comment applies to tpm2_nv{read,write}.

Reply:
Same response applies.